x/incentives/keeper/genesis.go

```diff
@@ -20,7 +19,7 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
 	}
 }
 
-// ExportGenesis returns the capability module's exported genesis.
+// ExportGenesis returns the incentives module's exported genesis.
```

Comment:
```suggestion
// ExportGenesis returns the x/incentives module's exported genesis.
```

Just to be consistent with other modules